static/js/containers/pages/DashboardPage.js

```diff
@@ -76,12 +76,16 @@ export class DashboardPage extends React.Component<DashboardPageProps, void> {
       <DocumentTitle title={`${SETTINGS.site_name} | ${DASHBOARD_PAGE_TITLE}`}>
         <div className="dashboard container">
           <h1>My Courses</h1>
-          <div className="enrolled-items container">
+          <div className="enrolled-items">
             {enrollments && enrollments.length > 0 ? (
               enrollments.map(this.renderEnrolledItemCard)
             ) : (
-              <div className="enrolled-item row card p-3 p-sm-5 rounded-0">
-                Once you enroll in a course, you can find it listed here.
+              <div className="placeholder-card p-3 p-sm-5 rounded-0">
```

Comment:
Your `.placeholder-card` styles just copy the styles for `.card` in `common.scss`. Can you get rid of those styles and change this approach to avoid the repetition?

```html
<div className="card no-enrollments p-3 p-sm-5 rounded-0">
  ...
</div>
```

```scss
// In dashboard.scss...

.enrolled-items {
  .card {
    display: flex;
    flex-direction: row;
    
    &.no-enrollments {
      flex-direction: column;
      
      a {
        color: black;
        text-decoration: underline;
      }
    }
    
```